technic/doc/chests.md

```diff
@@ -50,3 +50,11 @@ likely be revised.
 As with the chest of the basic Minetest game, each chest type comes
 in both locked and unlocked flavors.  All of the chests work with the
 pneumatic tubes of the pipeworks mod.
+
+## Sorting
+
+Chests can be sorted with one button with the following sort types : natural sort, by item, by quantity, by tipe or by wear. The chests can also be sorted automatically when its content changes.
```

Comment:
```suggestion
Chests can be sorted with one button with the following sort types : natural sort, by item, by quantity, by type or by wear. The chests can also be sorted automatically when its content changes.
```

Reply:
There are actually three buttons, one to change the sort method, one to sort, and one to toggle auto-sorting (except for iron chests, which lack auto-sorting).

Really the chest docs need a lot more changed, particularly when it comes to the mithril chests (which is what the "Chest digilines functionality" task is about).

Reply:
Additionally chests documentation should probably be moved under chests mod instead of keeping it in technic mod.
I think it would probably be better to create another PR just for chests documentations.